apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -62,17 +63,18 @@ export default class ModerationNotificationList extends Component {
     })
   }
 
-  getArchivedAlert = () => {
+  getSuccessAlert = (message) => {
+    const alertMessage = `Notification ${django.gettext(message)} successfully.`
```

Comment:
Sorry just saw this, so we run a translation on the `message` word and then we run a translation on the notification sentance with the `message` word in? is there a reason we don't just translate it once?

Reply:
@phillimorland  reason for this nested translation is that i then do not need to have to distinguish between them. which would be a big if/elseif/else block with four times almost the same sentence.

Reply:
ahh cool, but we need to remember to check it when we run translation, we have had issues with strings in js files not being picked up in the past.